recipes/nnn/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set build = 0 %}
+{% set version = "2.4" %}
+
+package:
+    name: nnn
+    version: {{version}}
+
+source:
+    url: https://github.com/jarun/nnn/archive/v{{version}}.tar.gz
+    sha256: 859ba5bd4892016aed9a4830ee1e1b03eb74e94c4f1bd82f0288dc559f7327eb
+
+build:
+    number: {{build}}
+
+requirements:
+    build:
+        - make
+        - {{ compiler('c') }}
+        - pkg-config
+        - ncurses
```

`ncurses` and `readline` should be `host` dependencies:
```suggestion
    host:
        - ncurses
```